### ISSUE TITLE
ktImportBLAST

### ISSUE THREAD

**Issue by arrusus** [marbl/Krona#112]
I'm trying to use ktImportBLAST to do a graph of my blasted NGS data. the problem is that I get not correct results. I expect to see just viruses ( I did my filtering so I'm sure of what I have in my blast file) and even if it is impossible, from krona I get a graph full of bacteria names (???) and no traces of my viruses (???).
 It is possible that my blast output is not set up correctly to be correctly read by krona?
 Before to make this trying I was sceptical because ktImportBLAST doesn't ask me to indicate where is the GI number or the evalue (or whatever) in the tab file, like it is able to locate each column inside the file. 
so basically the question is: there is a specific format required by krona to work properly ( for example the GI number is supposed to be somewhere specific, because I 've put it in the last column for example).

**Comment by arrusus** [comment]
to make the quesion more complete and clear I'll add the format used 
-task blastn -outfmt "6 qseqid qlen evalue bitscore score pident mismatch gaps ppos staxid ssciname scomname sblastname stitle sskingdom qstart qend sstart send sseqid sgi" -max_target_seqs 1

**Comment by ma-celik** [comment]
Hello have u found out anything? I am trying to do something similar too. Visualize my local blast results (metagenomics) on krona.

**Comment by MajaCN** [comment]
I am having the exact same issue, has anyone found a solution to this? I filtered out human host sequences from my samples, and my kraken output lists viral sequences (as expected in my study) but Krona gives me a graph of half bacteria and half "other root"...

**Comment by AlexSartori** [comment]
Yes, your BLAST outputs are malformed. In a standard BLAST tabular format, the second field is expected to be the subject id, while in your cases you have the query length (which Krona will mistakenly interpret as the subject ID).

Standard BLAST output for format 6: https://www.metagenomics.wiki/tools/blast/blastn-output-format-6
